src/UI/templates/default/Icon/icon.less

```diff
@@ -0,0 +1,67 @@
+.icon {
+	background-repeat: no-repeat;
+	background-size: cover;
+	background-image: url('./images/icon_default.svg');
```

Comment:
You miss sub-styles. There is an option to define a specific system style for only a part of ILIAS. If you want to exchange the icons of this sub-style this system breaks (you would have to edit the Less/CSS manually). 

Further, note that folder name "images" is variable it can be changed in the skins XML. I am not sure if many make use of this.